src/main/groovy/de/iteratec/osm/linechart/TimeSeries.groovy

```diff
@@ -0,0 +1,8 @@
+package de.iteratec.osm.linechart
```

Comment:
Probably a good idea not to re-use existing classes for chart, series and datapoints (_de.iteratec.osm.report.chart.OsmRickshawChart/OsmChartGraph/OsmChartPoint_) and istead starting with new ones for the angular timeseries board.